doc/kuadrantctl-ci-cd.md

```diff
@@ -86,15 +86,11 @@ spec:
     - name: run-kuadrantctl
       image: alpine:latest
       script: |
-        # Install yq silently
-        apk add --no-cache curl > /dev/null
-        curl -s -L https://github.com/mikefarah/yq/releases/download/v4.6.1/yq_linux_arm64 -o /usr/bin/yq > /dev/null && chmod +x /usr/bin/yq
-
         cd $(workspaces.source.path)
         mkdir -p generated-resources
-        ./kuadrantctl generate kuadrant authpolicy --oas openapi.yaml | yq eval -P | tee generated-resources/authpolicy.yaml
```

Comment:
Unrelated: Don't need to install `yq` anymore - `kuadrantctl` outputs generated resources in YAML format since 0.2.1